specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/CommonDefinitions.json

```diff
@@ -747,6 +747,182 @@
         }
       }
     },
+    "DaprResiliencyRetryPoliciesCollection": {
+      "description": "Dapr Resiliency Retry Policy ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DaprResiliencyRetryPolicy"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DaprResiliencyRetryPolicy": {
+      "description": "Resiliency Policy Retries",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "DaprResiliencyRetryPolicy resource specific properties",
+          "type": "object",
+          "properties": {
+            "name": {
+              "description": "policy name",
+              "type": "string"
```

Comment:
is this name different from the top level resource name ?

Reply:
Its the same, I will remove it.